bin/qpid_install.sh

```diff
@@ -7,7 +7,7 @@ sudo apt-get install -y libsasl2-2 libsasl2-dev
 
 # Get the latest Qpid Proton source
 cd $HOME/build
-git clone --branch 0.18.1 https://github.com/apache/qpid-proton.git
+git clone --branch 0.19.0 https://github.com/apache/qpid-proton.git
```

Comment:
What's the reason why we're not using the Rubygem version for this? https://rubygems.org/gems/qpid_proton

Reply:
We are https://github.com/ManageIQ/manageiq/pull/16793/files. But unlike most other gems `qpid_proton` gem is special (yeeey) because it depends on system library `qpid-proton`.

```
$ gem install qpid_proton
```

The command above will fail unless `qpid-proton` is installed on your system. In other words, `qpid_proton` is just a Ruby binding for C program called `qpid-proton`. To get things even better, `qpid-proton` program is poorly supported by install tools (apt-get, yum) because precompiled packages are not prepared yet in official repos (they are somehow stuck at stonage versions).

Reply:
Thanks @miha-plesko I knew there was a reason :)